types/braintree-web/venmo.d.ts

```diff
@@ -98,6 +98,10 @@ export function create(options: {
     ignoreHistoryChanges?: boolean | undefined;
     profileId?: string | undefined;
     deepLinkReturnUrl?: string | undefined;
+    allowDesktopWebLogin?: boolean | undefined;
+    allowDesktop?: boolean | undefined;
+    mobileWebFallBack?: boolean | undefined;
```

Comment:
The `| undefined` is redundant when the property is declared with `?`.

Reply:
Removed the redundant undefineds.